perf/dacapo/build.xml

```diff
@@ -16,18 +16,26 @@
 
 <project name="Dacapo suite" default="build" basedir=".">
 	<taskdef resource="net/sf/antcontrib/antlib.xml" />
-	<description>Dacapo suite</description>
+	<description>Dacapo Suite</description>
 
-	<!-- set global properties for this build -->
+	<!-- Set global properties for this build -->
 	<property name="DEST" value="${BUILD_ROOT}/perf/dacapo" />
-	<property name="src" location="." />
+	<property name="SRC" location="." />
 	
+	<condition property="isZOS" value="true">
+		<equals arg1="${os.name}" arg2="z/OS"/>
+	</condition>
+		
+	<condition property="git-prefix" value="git" else="https">
+		<isset property="isZOS"/>
```

Comment:
I am not sure `git-prefix` is needed on zos. The below downloading link is not a git repo link.

Reply:
Sorry, my bad! Thanks for noticing that.